src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs

```diff
@@ -406,15 +406,15 @@ public void SetSelectedPathOrNavigate(PathNavigationEventArgs e)
 					}
 				}
 			}
-			if (PathNormalization.NormalizePath(ParentShellPageInstance.FilesystemViewModel.WorkingDirectory) !=
-				PathNormalization.NormalizePath(e.ItemPath))
-			{
+
+			if (ParentShellPageInstance is null)
+				return;
+
+			if (NormalizePath(ParentShellPageInstance.FilesystemViewModel.WorkingDirectory) !=
```

Comment:
```suggestion
			if (NormalizePath(ParentShellPageInstance.FilesystemViewModel?.WorkingDirectory) !=
```

Reply:
Do you want to open a PR for this?

Reply:
I'm out of home right now so I can't open a PR right away. If you can, I will leave it to you.

Reply:
👍